.github/workflows/testing.yml

```diff
@@ -19,12 +19,16 @@ jobs:
             # run all combinations of the following, to make sure they're working together
             matrix:
                 # os: [ubuntu-latest, macos-latest, windows-latest]
-                php: [8.1, 8.2, 8.3]
+                php: ['8.1', '8.2', '8.3']
+                laravel: [^10.0, ^11.0]
                 dbal: [^3.0]
-                laravel: [10.*]
                 phpunit: [10.*]
-                dependency-version: [stable] # to add: lowest
-        
+                dependency-version: [stable] # to add: lowest 
+                exclude:
+                    -   laravel: "^11.0"
+                        php: "8.1"
+                        dbal: "^3.0"
```

Comment:
Hmmm something's fishy here, or I don't understand... looks like we're excluding L11 from testing? 👀 Why?

Reply:
Wait... no... We're excluding the L11 + PHP8.1 + DBAL 3.0 combo. Which shouldn't happen anyway. Got it!